src/components/sbb-datepicker/sbb-datepicker.tsx

```diff
@@ -0,0 +1,213 @@
+import {
+  Component,
+  ComponentInterface,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  JSX,
+  Listen,
+  Method,
+  Prop,
+  State,
+  Watch,
+} from '@stencil/core';
+import { NativeDateAdapter } from '../../global/helpers/native-date-adapter';
+import { getInput, InputUpdateEvent, isDateAvailable } from './sbb-datepicker.helper';
+import { AgnosticMutationObserver as MutationObserver } from '../../global/helpers/mutation-observer';
+import { documentLanguage, SbbLanguageChangeEvent } from '../../global/helpers/language';
+import { i18nDatePickerPlaceholder } from '../../global/i18n';
+
+const REGEX_PATTERN = /([0-9]{1,2})[.,\\/\-\s]?([0-9]{1,2})?[.,\\/\-\s]?([0-9]{1,4})?/;
+const REGEX =
+  /(^0?[1-9]?|[12]?[0-9]?|3?[01]?)[.,\\/\-\s](0?[1-9]?|1?[0-2]?)?[.,\\/\-\s](\d{1,4}$)?/;
+
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-datepicker.scss',
+  tag: 'sbb-datepicker',
+})
+export class SbbDatepicker implements ComponentInterface {
+  /** If set to true, two months are displayed */
+  @Prop() public wide = false;
```

Comment:
The idea was to handle every datepicker-related aspect on the datepicker itself, but to me it's the same. Let's see what @kyubisation has to say about this 😃

Reply:
For me it is better on the sbb-datepicker, as it is primarily related to the datepicker, not the toggle.

Reply:
On the other hand, the toggle has the calendar in it :) I can live also with this solution, I think it can be a little bit confusing. but also the other way :D